docs/management/gcache-write-set-cache-encryption.md

```diff
@@ -0,0 +1,100 @@
+# GCache and Write-Set cache encryption
```

Comment:
Just to be clear: 
1. GCache
2. Write-Set = Record-Set
These are two separate things

Reply:
Updated.